src/components/User-card.jsx

```diff
@@ -0,0 +1,5 @@
+const UserCard = () => {
```

Comment:
you should be sticking to only writing code that the ticket requires, you don't need these empty components yet